content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -168,212 +165,161 @@ To define the `Config` trait for the proof-of-existence pallet:
 Now that you've configured the pallet to emit events, you are ready to define those events.
 As described in [Design the application](#design-the-application), the proof-of-existence pallet emits an event under the following conditions:
 
-- When a new proof is added to the blockchain.
-- When a proof is revoked.
+- When a new claim is added to the blockchain.
+- When a claim is revoked.
 
 Each event also displays an `AccountId` to identify who triggered the
-event and the proof-of-existence data (as `Vec<u8>`) that is being stored or removed.
-By convention, each event includes an array with descriptive names for its parameters.
+event and the proof-of-existence claim (as `Hash`) that is being stored or removed.
 
 To implement the pallet events:
 
 1. Open the `pallets/template/src/lib.rs` file in a text editor.
 
 1. Replace the `#[pallet::event]` line with the following code block:
 
-   ```rust
-   // Pallets use events to inform users when important changes are made.
-   // Event documentation should end with an array that provides descriptive names for parameters.
-   #[pallet::event]
-   #[pallet::generate_deposit(pub(super) fn deposit_event)]
-   pub enum Event<T: Config> {
-   	/// Event emitted when a proof has been claimed. [who, claim]
-   	ClaimCreated(T::AccountId, Vec<u8>),
-   	/// Event emitted when a claim is revoked by the owner. [who, claim]
-   	ClaimRevoked(T::AccountId, Vec<u8>),
-   }
-   ```
+  ```rust
+  // Pallets use events to inform users when important changes are made.
+  // Event documentation should end with an array that provides descriptive names for parameters.
+  #[pallet::event]
+  #[pallet::generate_deposit(pub(super) fn deposit_event)]
+  pub enum Event<T: Config> {
+    /// Event emitted when a proof has been claimed.
+    ClaimCreated { who: T::AccountId, claim: T::Hash },
+    /// Event emitted when a claim is revoked by the owner.
+    ClaimRevoked { who: T::AccountId, claim: T::Hash },
```

Comment:
named fields will populate the info through the metadata, so we will know exactly what data means